.github/workflows/publish-helm-chart.yaml

```diff
@@ -20,14 +22,17 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@v1
         with:
-          version: v3.5.0
+          version: v3.6.0
       - name: Install chart-releaser
+        env:
+          VERSION: 1.2.1
         run: |
-          wget https://github.com/helm/chart-releaser/releases/download/v1.1.1/chart-releaser_1.1.1_linux_amd64.tar.gz
-          tar xzvf chart-releaser_1.1.1_linux_amd64.tar.gz cr
+          wget "https://github.com/helm/chart-releaser/releases/download/v${VERSION}/chart-releaser_${VERSION}_linux_amd64.tar.gz"
+          tar xzvf chart-releaser_${VERSION}_linux_amd64.tar.gz cr
       - name: Package helm chart
         run: |
-          ./cr package ${{ env.CHART_DIR }}
+          RELEASE=$(echo ${{ github.ref }} | sed -e "s#refs/tags/##g" | sed -e 's/^v//')
+          helm package --app-version=${RELEASE} --version=${RELEASE} ${{ env.CHART_DIR }} -d .cr-release-packages
```

Comment:
If we take version from git revision how and when will we update versions in Chart.yaml https://github.com/aquasecurity/starboard/blob/main/deploy/helm/Chart.yaml#L9 and https://github.com/aquasecurity/starboard/blob/main/deploy/helm/Chart.yaml#L14

For now, before each release I update Chart.yaml and then trigger this build so even if it's kinda manual the value is src code is consistent with the version of published chart.

Reply:
In this PR, the chart version and the chart app version will be the same. I will use the simple 1-1 versioning. I tried to follow this approach: https://codefresh.io/docs/docs/new-helm/helm-best-practices/